src/MCPServer/install/README.md

```diff
@@ -122,6 +122,18 @@ This is the full list of variables supported by MCPServer:
     - **Type:** `boolean`
     - **Default:** `true`
 
+- **`ARCHIVEMATICA_MCPSERVER_MCPSERVER_STORAGE_SERVICE_CLIENT_TIMEOUT`**:
+    - **Description:** configures the Storage Service client to stop waiting for a response after a given number of seconds.
+    - **Config file example:** `MCPClient.storage_service_client_timeout`
+    - **Type:** `float`
+    - **Default:** `86400`
+
+- **`ARCHIVEMATICA_MCPSERVER_MCPSERVER_STORAGE_SERVICE_CLIENT_QUICK_TIMEOUT`**:
+    - **Description:** configures the Storage Service client to stop waiting for a response after a given number of seconds. This applies only to requests that are supposed to return immediately.
+    - **Config file example:** `MCPClient.storage_service_client_timeout`
+    - **Type:** `float`
+    - **Default:** `5`
```

Comment:
The new `package` module in MCPServer responsible for starting transfers/packages talks to Storage Service. As we had never talked to Storage Service from MCPServer before we did not have settings. The `storageService` module does not provide defaults so we have to provide them instead.